### ISSUE TITLE
No games using Proton work

### ISSUE THREAD

**Issue by PureTryOut** [flathub/com.valvesoftware.Steam#552]
**Game information**
Multiple games, including GTA5, Magicka and Risk: Global Domination.

**Distribution name and version where applicable**
```
▶ cat /etc/os-release 
NAME=Gentoo
ID=gentoo
PRETTY_NAME="Gentoo/Linux"
ANSI_COLOR="1;32"
HOME_URL="https://www.gentoo.org/"
SUPPORT_URL="https://www.gentoo.org/support/"
BUG_REPORT_URL="https://bugs.gentoo.org/"
```

**Flatpak info**
```
▶ flatpak --version
Flatpak 1.4.4
                                                                                                                          
▶ flatpak --gl-drivers
default
host
```

**Problem description**
Games that according to ProtonDB work as Gold or even Platinum just fail to launch for me. This includes for example Grand Theft Auto 5, Magicka and Risk: Global Domination. In case of GTA5 it tells me to check the game data, in case of Risk: Global Domination it just closes/crashes after the initial loading popup, and in case of Magica it crashes without ever showing any window.

**Does this issue reproduce with native Steam**
Have not tested and I can not since this is a purely 64-bit system.

**Comment by ajoneil** [comment]
Seeing the same behaviour here on Arch Linux - games that natively support Linux work, but proton games do not.

**Comment by TjayNL** [comment]
I have kinda the same issue after adding an external drive ALL the proton games don't seem to work anymore.

**Comment by nanonyme** [comment]
Closing as dupe of https://github.com/flathub/com.valvesoftware.Steam/issues/642

**Comment by gasinvein** [comment]
@nanonyme I don't think if it's the same issue? Proton 5.13 was released in October 2020, while this one was reported long before that, in April.

**Comment by nanonyme** [comment]
Good point. Can someone test with new Proton if this works correctly there?
